pkg/server/server.go

```diff
@@ -28,9 +32,17 @@ func NewServer(apiGroups ...srvtypes.APIGroup) *Server {
 		versionedAPIs = append(versionedAPIs, apiGroup.VersionedAPIs()...)
 	}
 
+	srvMetrics := grpcprom.NewServerMetrics(
```

Comment:
Here's an example of setting metrics through component-base/metrics, would it be possible to reuse that code? 

https://github.com/kubernetes/kubernetes/blob/dc3021b156a6a5adc4b4898d366822529bcc74a4/pkg/scheduler/metrics/metrics.go#L149

Reply:
Sure. Moved all the metrics related codes to a single file.